addons/hr/static/src/views/open_chat_hook.js

```diff
@@ -8,11 +8,14 @@ patch(helpers, {
         ...helpers.SUPPORTED_M2X_AVATAR_MODELS,
         "hr.employee",
         "hr.employee.public",
+        "hr.candidate",
     ],
     buildOpenChatParams(resModel, id) {
-        if (["hr.employee", "hr.employee.public"].includes(resModel)) {
+        if (resModel === "hr.candidate") {
```

Comment:
Studio has nothing to do with this.
Test should be a unit test in HR, not relying at all on studio.